app/src/pages/Signup.tsx

```diff
@@ -1,116 +1,271 @@
-            <input
-              type="text"
-              className="rounded border flex-1 appearance-none border-gray-300 w-full py-2 px-4 bg-white text-gray-700 placeholder-gray-400 shadow-sm text-base focus:outline-none focus:ring-2 focus:ring-blue-600 focus:border-transparent"
-              placeholder="Last Name"
-              {...register("lastName", { required: "Please enter your last name." })}
-            />
-            {errors.lastName?.message}
-
-            <label className="font-semibold text-xs mt-3" htmlFor="usernameField">Email</label>
-            <input
-              type="text"
-              className="rounded border flex-1 appearance-none border border-gray-300 w-full py-2 px-4 bg-white text-gray-700 placeholder-gray-400 shadow-sm text-base focus:outline-none focus:ring-2 focus:ring-blue-500 focus:border-transparent"
-              placeholder="Email"
-              {...register("email", { required: "Please enter your email." })}
-            />
-            {errors.email?.message}
-
-            <label className="font-semibold text-xs mt-3">Motivation box</label>
-            <textarea
-              className="flex-1 appearance-none border border-gray-300 w-full py-2 px-4 bg-white text-gray-700 placeholder-gray-400 rounded text-base focus:outline-none focus:ring-2 focus:ring-blue-500 focus:border-transparent resize-none"
-              id="motivation"
-              placeholder="What is your motivation for applying as a content creator?"
-              rows={5}
-              {...register("motivation", { required: "Please enter a motivation." })}
-            ></textarea>
-            <p>{errors.motivation?.message}</p>
-
-            <button
-              className="flex items-center justify-center h-12 px-6 w-full bg-blue-600 mt-8 rounded font-semibold text-sm text-blue-100 hover:bg-blue-700"
-              type='submit'
-            >
-              Submit
-            </button>
-          </form>
-        </div>
+  const [isFormValid, setIsFormValid] = useState(false);
+  
+  // Variables determining whether or not the password is visible
+  const [passwordVisible, setPasswordVisible] = useState(false);
+  const togglePasswordVisibility = () => {
+    setPasswordVisible(!passwordVisible);
+  }
+  const [passwordVisibleRepeat, setPasswordVisibleRepeat] = useState(false);
+  const togglePasswordVisibilityRepeat = () => {
+    setPasswordVisibleRepeat(!passwordVisibleRepeat)
+  }
+
+  //Variables and functions for checking and setting password checks
+  const [passwordCheck1, setPasswordCheck1] = useState(false);
+  const [passwordCheck2, setPasswordCheck2] = useState(false);
+
+  const handlePasswordChange = (e: React.FocusEvent<HTMLInputElement>) => {
+    const password = e.target.value;
+  
+    const isCheck1Fulfilled = password.length >= 8;
+    setPasswordCheck1(isCheck1Fulfilled);
+  
+    const isCheck2Fulfilled = /[a-aZ-z]/.test(password);
```

Comment:
The one in educado-mobile*:
```javascript
/**
  * Checks if the password contains at least one letter
  * @param {String} password
  * @returns {Boolean} true if password contains at least one letter, false otherwise
  */
const validatePasswordContainsLetter = (password) => {
  const regex = /.*\p{L}.*$/u;
  return regex.test(password);
}
```

Reply:
Regex has been fixed